projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -408,6 +408,7 @@ if (environment.cpq) {
         unifiedDefaultHeaderSlotsAcrossBreakpoints: true,
         reserveSpaceForImagesOnPdpAndPlp: true,
         lazyLoadImagesByDefault: true,
+        incrementProcessesCountForMergeCart: true,
```

Comment:
is it not JDK21 related? Maybe it should be below the comment

Reply:
According to the above comment, it is not related to JDK 21 and it is not necessary to keep it below the comment.